src/Service/UserService.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: backin
+ * Date: 04/02/2019
+ * Time: 18:21
+ */
```

Comment:
Try to delete those lines when they're generated, or turn off the automatic generation. Not blocking for this PR though.